app/components/App/App.js

```diff
@@ -0,0 +1,207 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { autobind } from 'core-decorators';
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
+import findPath from '../../utils/BFS';
+import Game from '../Game/Game';
+// const Settings = require('./Settings');
+// const Results = require('./Results');
+
+function resetBoard(lengthX, lengthY) {
+  console.log('creating board');
```

Comment:
creating or resetting? I'd say this function creates a board, because it doesn't accept one as an argument to reset.

Reply:
done